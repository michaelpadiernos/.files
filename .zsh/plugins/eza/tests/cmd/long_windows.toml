bin.name = "eza"
args = "tests/itest --long --no-user --no-time --no-filesize"
