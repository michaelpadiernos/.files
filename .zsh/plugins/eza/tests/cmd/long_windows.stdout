-a--- a
-a--- b
-a--- c
-a--- d
-a--- e
d---- exa
-a--- f
-a--- g
-a--- h
-a--- i
-a--- image.jpg.img.c.rs.log.png
-a--- index.svg
-a--- j
-a--- k
-a--- l
-a--- m
-a--- n
-a--- o
-a--- p
-a--- q
d---- vagrant
